service/controller/v2/arm_templates/main.json

```diff
@@ -21,6 +21,12 @@
         "description": "Whether the deployment is provisioned the very first time."
       }
     },
+    "masterVersionBundleVersions":{
+      "type":"object",
+      "metadata":{
+        "description":"JSON blob of key-value pairs mapping master VMSS instance names to their applied version bundle versions."
+      }
+    },
```

Comment:
This is part of the hack I have to backport so we can transport version bundle version information from `v2` to `v3`. I tested this and it looks fine. Existing clusters het the versions applied automatically and once we update them later on the update will be fine because we have the version information prepared. So the trick now is that we set the version blobs to the `master-vmss-deploy` and `worker-vmss-deploy` deployments. These exist already and also exist in `v3`. Only difference is that they are managed in another operatorkit resource now.